.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove the folder `.vs` and add `.vs` to the `.gitignore` file so we don't commit these files by mistake